src/shared/components/comment/comment-node.tsx

```diff
@@ -326,10 +328,20 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
               )}
               {locked && (
                 <span
-                  className="unselectable pointer mx-1"
+                  className="mx-1"
                   data-tippy-content={I18NextService.i18n.t("locked")}
                 >
-                  <Icon icon="lock" classes="icon-inline text-danger" />
+                  <Icon icon="lock" classes="icon-inline" />
+                </span>
+              )}
+              {(deleted || removed) && (
+                <span
+                  className="mx-1"
+                  data-tippy-content={I18NextService.i18n.t(
+                    deleted ? "deleted" : "removed",
+                  )}
+                >
+                  <Icon icon="trash" classes="icon-inline" />
```

Comment:
I believe we use the `x` icon with `text-danger` for removed, so this might be a little cleaner as two separate blocks.

Up to you though